include/exadg/structure/material/library/st_venant_kirchhoff.cpp

```diff
@@ -94,10 +98,10 @@ StVenantKirchhoff<dim, Number>::get_f2_factor() const
   Number const nu           = data.nu;
   Type2D const type_two_dim = data.type_two_dim;
 
-  return (dim == 3) ? (1. - 2. * nu) / 2. / (1. + nu) / (1. - 2. * nu) :
+  return (dim == 3) ? (1. - 2. * nu) * 0.5 / ((1. + nu) * (1. - 2. * nu)) :
                       (type_two_dim == Type2D::PlaneStress ?
-                         ((1. - nu) / 2. / (1. - nu * nu)) :
-                         ((1. - 2. * nu) / 2. / (1. + nu) / (1. - 2. * nu)));
+                         ((1. - nu) * 0.5 / (1. - nu * nu)) :
+                         ((1. - 2. * nu) * 0.5 / ((1. + nu) * (1. - 2. * nu))));
```

Comment:
only a minor comment, since you made these changes: one could write the factor of two inside the (new) brackets. Could be easier to read, but is probably also a matter of taste.